styles/settings/font_family_map.scss

```diff
@@ -0,0 +1,3 @@
+// Font Family
+$font-families: ('sans': ("Sole Sans", Helvetica, sans-serif),
+    'extended': ("Sole Sans Extended", Helvetica, sans-serif));
```

Comment:
There are a lot of SCSS linting errors. Most of them are auto-fixable , so you can run `npm run lint:scss -- --fix`. Then check the output for any remaining issues and you can fix those manually.

Reply:
Made the font and linting changes! :)